plugins/woocommerce-admin/client/customize-store/intro/intro.scss

```diff
@@ -109,6 +109,14 @@
 	width: 100%;
 	align-items: center;
 
+	.ai-disclaimer {
```

Comment:
If you move this to L194 inside the `.woocommerce-customize-store-banner-content p` rule and change the selector to `&.ai-disclaimer`, we won't need to use `!important` here, which may result in unexpected overrides in the future

Reply:
Updated in 3ea047f0a567348686aa2095b3e8581f011d02b3